config/config_test.go

```diff
@@ -264,6 +278,17 @@ func TestGetDefaultServiceTTL(t *testing.T) {
 	}
 }
 
+func TestGetDefaultLayersTTL(t *testing.T) {
```

Comment:
It is not the default value. Change either test name or remove value from test config.